docs/tutorials.rst

```diff
@@ -38,6 +38,11 @@ Notebooks
    notebooks/sed_fitting_gammacat_fermi.ipynb
    notebooks/fermi_lat.ipynb
    notebooks/light_curve.ipynb
+   notebooks/cta_sensitivity.ipynb
+   notebooks/nddata_demo.ipynb
```

Comment:
nddata_demo is showing a low-level class for Gammapy devs. It should not be shown to users.

Ideally we would still keep / maintain it in the Gammapy repo, but configure the docs build and notebook download to not expose it to users.

This is possible, no?

Reply:
Ok.
nddata_demo is referenced from the docs.
http://docs.gammapy.org/dev/api/gammapy.utils.nddata.NDDataArray.html?highlight=nddata_demo
It has to be built in the docs to avoid Sphinx warnings

I can remove from the toc in tutorials
Should I remove it also from the `notebooks.yaml`?
https://github.com/gammapy/gammapy/blob/4eba425bb033b896ff2852d11b8974a0610df0fb/tutorials/notebooks.yaml#L69

Reply:
> Should I remove it also from the notebooks.yaml?

I'm not sure.